go.mod

```diff
@@ -243,7 +243,7 @@ require (
 	github.com/dave/dst v0.27.2 // @grafana/grafana-as-code
 	github.com/go-jose/go-jose/v3 v3.0.3 // @grafana/identity-access-team
 	github.com/grafana/dataplane/examples v0.0.1 // @grafana/observability-metrics
-	github.com/grafana/dataplane/sdata v0.0.7 // @grafana/observability-metrics
+	github.com/grafana/dataplane/sdata v0.0.8-0.20240409212435-4bd1afeef43f // @grafana/observability-metrics
```

Comment:
This will be updated after https://github.com/grafana/dataplane/pull/41 is merged and a new version of `datplane/sdata` is available